clang-tools-extra/clang-tidy/readability/RedundantInlineSpecifierCheck.cpp

```diff
@@ -0,0 +1,99 @@
+//===--- RedundantInlineSpecifierCheck.cpp -
+// clang-tidy----------------------===//
```

Comment:
Removing 3 `-` should make the formatter happy and making it look more like the rest of our comment.
```suggestion
//===--- RedundantInlineSpecifierCheck.cpp - clang-tidy -------------------===//
```